### PR TITLE
Created  1. Introduction to The Linux Foundation.md

### PR DESCRIPTION
**Pull request by Vrashabh-Sontakke**
Initiated LINUX_Notes_(LFS101) Project.

## Changes proposed

Added LINUX_Notes_(LFS101) folder and sub-folders

## Note to reviewers

Notes are neither too lengthy nor too short, they are sweetly adequate.

### CODE REVIEW

**Review by bishal7679 — COMMENTED** [review]
@Vrashabh-Sontakke  You should always open an issue & then make the necessary PR for that.
Please follow this approach ..
I hope you understand 👍 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by Vrashabh-Sontakke** [comment]
> @Vrashabh-Sontakke  You should always open an issue & then make the necessary PR for that.
> Please follow this approach ..
> I hope you understand 👍 
> 
> __
> [sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix
> 
> Oh, actually this is my first ever PR. I would definitely follow that from now on.
Thanks for the help.

**Comment by Vrashabh-Sontakke** [comment]
created an issue #89  with reference to this PR #88

**Comment by Vrashabh-Sontakke** [comment]
> @Vrashabh-Sontakke You should always open an issue & then make the necessary PR for that. Please follow this approach .. I hope you understand +1
> 
> __ [sema-logo](https://semasoftware.com/gh) [ sema-logo ](https://semasoftware.com/gh) [ ](https://semasoftware.com/gh)  **Summary:** hammer_and_wrench This code needs a fix

I created an issue #89 and closed it as resolved. Could we link this PR #88 with the issue #89.

File: docs/LINUX_Notes_(LFS101)/1_Introduction to The Linux Foundation/1. Introduction to The Linux Foundation.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+**Introduction** 
+
+The kernel is just the core of the OS, it talks to the HW, makes the HW works, and it makes it able for you to run a program.
```
Comment (kranurag7) [inline docs/LINUX_Notes_(LFS101)/1_Introduction to The Linux Foundation/1. Introduction to The Linux Foundation.md:3]:
```suggestion
#### What is Linux Kernel? 
The kernel is just the core of the OS, it talks to the hardware, makes the hardware works, and it makes it able for you to run a program.
```

File: docs/LINUX_Notes_(LFS101)/1_Introduction to The Linux Foundation/1. Introduction to The Linux Foundation.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+**The LINUX Foundation** 
+
+It's a non-profit organization that sponsors Linux
```
Comment (kranurag7) [inline docs/LINUX_Notes_(LFS101)/1_Introduction to The Linux Foundation/1. Introduction to The Linux Foundation.md:7]:
```suggestion
It's a non-profit organization that sponsors Linux and various other open source projects. 
```

File: docs/LINUX_Notes_(LFS101)/1_Introduction to The Linux Foundation/1. Introduction to The Linux Foundation.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
+**Introduction** 
+
+The kernel is just the core of the OS, it talks to the HW, makes the HW works, and it makes it able for you to run a program.
+
+**The LINUX Foundation**
```
Comment (kranurag7) [inline docs/LINUX_Notes_(LFS101)/1_Introduction to The Linux Foundation/1. Introduction to The Linux Foundation.md:5]:
```suggestion
**The Linux Foundation** 
```
```suggestion
**The Linux Foundation** 
```

File: docs/LINUX_Notes_(LFS101)/1_Introduction to The Linux Foundation/1. Introduction to The Linux Foundation.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
...hCTfuWnZr9gFmAGWtGYgxg+KbuSR5BCQIiKEGqChyBIPaoXFMYhBAkFqNHsOTCQmCC1QTatoYhElRNogrGIRPlIozzwP7Cp6ZQ+zJ+k+iW5ebbT/wB5nkUJWJOq1XivorDFB945mf8AKGmYDUfSenlEAobJKkbg+33IXw8B7BpXRB8iPADdDuIRwknG/wBAIECBAgQIECBAgQIECBEpIZMEJXGNvKONxGgU+Ufw17uf99JAIgjkaSkRVmF2J/3yt0TjEJaCP7fQCBAgQIECBAgQIECBAgvMAHMpVX+vs4Gcm5+vqHa8WImuvuZf+UAAAFgNPTubABk8NMU4tYefbX72BmJ/t/TvcWBqGwdaipjMy+fVAK2TGNPYx/tIBBEkTD9673v9V7mqcLBYgLgfj0lq1atWrVq1aZ+pTyuSLJ08y1atWrSI1QmqCxIGB+PCZQ8aYAlbeZatWrVp4qEXhoN8JV9oS4THorVq1atWrVrcwSqAG/3F3vf6r4/Rd87Hv+g73u+4973+q+Pl61aZyKQcWIONTvB+zRlTRGFZibTLKkQUhFQRbkoSVLoY0SFxwFUFBcoUGsl0JOD6IgDY2qP+hrgYHBViy6z0ygL+UNyhgjz/ADse/wAgfRZLIiGa0kUQppmDZMTNEh5CsF6IjLI+QcAVEYHasFwEwhWACT3rTFIkQyFYLivYRNZtDuRF4iTNGZ4W2/IWm8pJqaX5aJ4kZWSwhks1YK87GNgy3cC2bk466IudylmNZDL4973fce976bbqfH/XqY+XlfCQuGkcSZOVCHPnXYw3QtqzBe1n4M7AQsoCGBtSltoQohLEl6RmLbEtQAlTSI4ghJRXNHh1otHUZhtToDStpWItwUgo4xnz/Ox7/JPyhv4ykUvERhpqkggClGA3RiIAmaQJnruQsJBsXQlG9d5CojZxSxIZZq5MXjEpBQBRJFqsFtxcGQSZE0hQEEEwEgVgyQyBDV2/5sgGSFgyzzqciLhgDpZDIgYjJN0WLMWoQJu4uBfHve77ipOfnFQIOm9h/fqY/Rd87Hv+g73u+4gRXEhqa7aMaX7L/FQR0r3pnhw9O5qnCQSZQ4X5+gWrVq1atWrVq1atWrVrEaqzRRJhDC/PhMoedEiQlvoFq1atWrVq1atWrVq1a3MEIoJt9yQ7yAJWcMmGOlBXQSmInMhv/wDC/wD/2Q==)
```
Comment (kranurag7) [inline docs/LINUX_Notes_(LFS101)/1_Introduction to The Linux Foundation/1. Introduction to The Linux Foundation.md:47]:
```suggestion

```

**Review by kranurag7 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**